app_flutter/lib/service/dev_cocoon.dart

```diff
@@ -171,19 +171,24 @@ class DevelopmentCocoonService implements CocoonService {
     assert(age >= 0);
```

Comment:
Observation: The fake data is showing attempts at 0 for a lot of the task statuses.

Reply:
Updated this logic to have more plausible attempt counts.